src/app/components/AllowanceMenu/index.tsx

```diff
@@ -138,14 +143,35 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
                 onChange={(e) => setBudget(e.target.value)}
               />
             </div>
+            <Setting
+              title={t("enable_login.title")}
+              subtitle={
+                <Trans
```

Comment:
Now that the string was changed do we still need the `Trans` here?

Reply:
Oops! Changing it

Reply:
Done!